src/av2/utils/io.py

```diff
@@ -206,7 +206,7 @@ def write_img(img_path: Path, img: NDArrayByte, channel_order: str = "RGB") -> N
     cv2.imwrite(str(img_path), img)
 
 
-def read_json_file(fpath: Path) -> Any:
+def read_json_file(fpath: Path) -> Any:  # noqa: ANN401
```

Comment:
This has been fixed in `main`. Perhaps we could pull the changes into this branch?

Reply:
Got it! Will do.